src/h3lib/include/linkedGeo.h

```diff
@@ -24,8 +24,24 @@
 #include "geoCoord.h"
 #include "h3api.h"
 
-void initLinkedPolygon(LinkedGeoPolygon* polygon);
-void initLinkedLoop(LinkedGeoLoop* loop);
+/**
+ * Initialize a linked polygon
+ * @param  polygon Polygon to init
+ */
+#define initLinkedPolygon(polygon)          \
```

Comment:
Why not something like the existing `H3_INIT`, so in code `*polygon = LINKED_GEO_POLYGON_INIT;`
Please make macros ALL_CAPS.

I assume this is for performance reasons. I'd suggest that performance changes should have associated benchmarking.

Reply:
Ya sure. Actually not really for performance. I just think this is easy enough to be a one-liner, not a separate function. Might try removing entirely.